code/go/0chain.net/smartcontract/dbs/event/blobber_test.go

```diff
@@ -287,42 +290,104 @@ func TestBlobberLatLong(t *testing.T) {
 func TestBlobberGetCount(t *testing.T) {
 	access := config.DbAccess{
 		Enabled:         true,
-		Name:            os.Getenv("POSTGRES_DB"),
-		User:            os.Getenv("POSTGRES_USER"),
-		Password:        os.Getenv("POSTGRES_PASSWORD"),
-		Host:            os.Getenv("POSTGRES_HOST"),
-		Port:            os.Getenv("POSTGRES_PORT"),
+		Name:            "events_db",
+		User:            "zchain_user",
+		Password:        "zchian",
+		Host:            "localhost",
+		Port:            "5432",
```

Comment:
Please avoid exposing db username and pass, hard-coded data makes it unable to change if the tests evn use different username/pass